File: src/main/java/com/github/graqr/threshr/model/redsky/products/Hold.java
package com.github.graqr.threshr.model.redsky.products;

import com.fasterxml.jackson.annotation.JsonProperty;
import io.micronaut.serde.annotation.Serdeable;

@Serdeable
public record Hold(@JsonProperty("is_active") Boolean isActive) {
}

File: src/main/kotlin/com/graqr/threshr/circleStore/model/product/Hold.kt
package com.graqr.threshr.circleStore.model.product

import com.fasterxml.jackson.annotation.JsonProperty
import io.micronaut.serde.annotation.Serdeable

@Serdeable
data class Hold(
    @param:JsonProperty("is_active")
    val isActive: Boolean) {
}
